scripts/migration/migrate_date_modified_for_existing_nodes.py

```diff
@@ -25,7 +26,7 @@ def main():
 
     logger.warn('Date_modified field will be added to all nodes.')
 
-    for node in models.Node.find():
+    for node in models.Node.find(Q('date_modified', 'eq', None)):
```

Comment:
But did you create those nodes after the `date_modified` field was added to the codebase? What about nodes that were created before the field was added?